examples/bastion/config_no_pub_ip.json

```diff
@@ -30,6 +33,7 @@
             "public_ip": false,
             "image": "variables.jumpbox_image",
             "accelerated_networking": true,
+            "cloud_init_sasurl": "variables.cloud_init_sasurl",
```

Comment:
This doesn't need to be a sasurl - any url is fine, so it might be better to just name cloud_init or cloud_init_url?

Reply:
I changed the cloud_init keyword to cloud_init_url and updated the changelog.